src/mainboard/pcengines/apu2/mainboard.c

```diff
@@ -45,6 +45,12 @@
 #endif //CONFIG_USE_CBMEM_FILE_OVERRIDE
 #include <cbfs_core.h>
 #include <spd_cache.h>
+#include <smbios.h>
+#include <string.h>
+
+#define NB_CFG_MSR (0xC001001f)
+// 46th bit of NB_CFG_MSR register
+#define ENABLE_CF8_EXT_CFG (1<<(46-32))
```

Comment:
@macpijan same thing [here](https://github.com/pcengines/coreboot/blob/apu2b-20160304/3rdparty/pi/amd/00730F01/Proc/CPU/cpuRegisters.h#L285). Also for future: never leave thing like `46` or `32` in such macro without other define or extensive description.